metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -211,7 +219,6 @@ final class BloopServers(
       bloopCreatedByMetalsFilePath: AbsolutePath,
       maybeBloopJvmProperties: Option[List[String]],
       maybeJavaHome: Option[String],
-      bloopJsonUpdateCause: BloopJsonUpdateCause,
```

Comment:
why is this removed? This was used for telling the user why bloop was to be restarted, especially when it was caused by changing Metals java home.

Reply:
Same as above.